kolibri/utils/cli.py

```diff
@@ -399,21 +398,60 @@ def plugin(plugin_name, **args):
             if _is_plugin(obj):
                 plugin_classes.append(obj)
     except ImportError as e:
-        if e.message.startswith("No module named"):
+        if str(e).startswith("No module named"):
             raise PluginDoesNotExist(
                 "Plugin '{}' does not seem to exist. Is it on the PYTHONPATH?".
                 format(plugin_name)
             )
         else:
             raise
 
+    if not plugin_classes:
+        # There's no clear use case for a plugin without a KolibriPluginBase
+        # inheritor, for now just throw a warning
+        logger.warning(
+            "Plugin '{}' has no KolibriPluginBase defined".format(plugin_name)
+        )
+
+    return plugin_classes
+
+
+def plugin(plugin_name, **args):
+    """
+    Receives a plugin identifier and tries to load its main class. Calls class
+    functions.
+    """
+    from kolibri.utils import conf
+
     if args.get('enable', False):
+        plugin_classes = get_kolibri_plugin(plugin_name)
         for klass in plugin_classes:
             klass.enable()
 
     if args.get('disable', False):
-        for klass in plugin_classes:
-            klass.disable()
+        try:
+            plugin_classes = get_kolibri_plugin(plugin_name)
+            for klass in plugin_classes:
+                klass.disable()
+        except PluginDoesNotExist as e:
+            logger.error(str(e))
+            logger.warning(
+                "Removing '{}' from configuration in a naive way.".format(
+                    plugin_name
+                )
+            )
+            module_found = False
+            for module_path in conf.config['INSTALLED_APPS']:
+                if module_path.startswith(plugin_name):
```

Comment:
Could we make this an exact equals? It will be a bit more of a burden for a user if they are trying to import a bunch of plugins from the same (non-existent) parent module, but it is safer in that it will only remove the plugin that they asked to remove.